setup.py

```diff
@@ -4,20 +4,20 @@
 from setuptools import setup, find_packages
 
 
-# NOTE: If adding requirements make sure to also add to requirements-dev.txt
+# NOTE: If adding requirements make sure to also add to Pipfile
 # NOTE: When updating botocore make sure to update awscli/boto3 versions below
 install_requires = [
     # pegged to also match items in `extras_require`
     'botocore>=1.15.3,<1.15.16',
     'aiohttp>=3.3.1',
     'wrapt>=1.10.10',
-    'async_generator>=1.10',  # can remove if we move to py3.6+
+    'aioitertools>=0.5.1',
 ]
 
 PY_VER = sys.version_info
 
-if not PY_VER >= (3, 5, 3):
-    raise RuntimeError("aiobotocore doesn't support Python earlier than 3.5.3")
+if not PY_VER >= (3, 6, 0):
+    raise RuntimeError("aiobotocore doesn't support Python earlier than 3.6.0")
```

Comment:
So https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires exists, would using that be better?

Reply:
great idea: https://github.com/aio-libs/aiobotocore/pull/769/commits/965700b457556be01b2e06c806e53c9b8149af0a